conans/test/unittests/client/generators/premake_test.py

```diff
@@ -36,8 +36,8 @@ class PremakeGeneratorTest(unittest.TestCase):
     conan_libdirs_MyPkg1 = {{"{lib1}"}}
     conan_bindirs_MyPkg1 = {{"{bin1}"}}
     conan_libs_MyPkg1 = {{"libfoo"}}
-    conan_cppdefines_MyPkg1 = {{"MYDEFINE1"}}
-    conan_cppflags_MyPkg1 = {{"-fPIE"}}
+    conan_defines_MyPkg1 = {{"MYDEFINE1"}}
+    conan_cxxflags_MyPkg1 = {{"-fPIE"}}
```

Comment:
```suggestion
    conan_cppflags_MyPkg1 = {{"-fPIE"}}
```

Reply:
Finally we agreed on having ``cpp_info.cppflags`` converted to ``conan_cxxflags``. So no further action is required.